Classes/Indexer/AbstractNodeIndexer.php

```diff
@@ -91,6 +92,8 @@ protected function evaluateEelExpression($expression, NodeInterface $node, $prop
      * @param string $fulltextExtractionExpression
      * @param array $fulltextIndexOfNode
      * @throws IndexingException
+     * @throws \Neos\ContentRepository\Exception\NodeException
+     * @throws \Neos\Eel\Exception
```

Comment:
These exceptions are thrown in called methods - we only annotate the exceptions thrown here (maybe you should reconfigure your phpStorm :))

Reply:
> we only annotate the exceptions thrown here

Well, _do we_?  Says who?

> maybe you should reconfigure your phpStorm

Actually those warnings bother me, but I don't want to  disable the inspection (since on actually `throw`ing it definitely must be added). So I started to add them also when it's an "indirect throw", and found it quite helpful.

Anyway, let's handle that in https://discuss.neos.io/t/to-throws-or-not-to-throws-indirect-exceptions/3018 – ok?